drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -459,6 +459,53 @@ static void set_up_plls(void)
 	}
 	LL_RCC_PLL_Disable();
 
+#if defined(STM32_PLL2_ENABLED)
```

Comment:
IIC this should be done outside of `#if defined(STM32_PLL_ENABLED)` as PLL2 could be enabled even if PLL is not enabled.